includes/rest/class-collection.php

```diff
@@ -56,6 +57,19 @@ public static function register_routes() {
 				),
 			)
 		);
+
+		\register_rest_route(
+			ACTIVITYPUB_REST_NAMESPACE,
+			'/users/(?P<user_id>[\w\-\.]+)/collections/moderators',
```

Comment:
Do we really even need to bother with the user ID here? Since it's only ever for the Blog User, we could just use `/collections/moderators` or even just `/moderators` ?

Reply:
I will think about that... good point!